services/wfm/src/main/java/org/openkilda/wfm/topology/network/service/NetworkWatcherService.java

```diff
@@ -93,13 +93,17 @@ public void tick() {
     }
 
     /**
-     * .
+     * Process discovery packet confirmation.
      */
-    public void confirmation(Endpoint endpoint, long packetNo) {
+    public void processConfirmation(Endpoint endpoint, long packetNo, boolean confirmed) {
         log.debug("Watcher service receive SEND-confirmation for {} id:{} task:{}", endpoint, packetNo, taskId);
         Packet packet = Packet.of(endpoint, packetNo);
         if (producedPackets.remove(packet)) {
-            confirmedPackets.add(packet);
+            if (confirmed) {
+                confirmedPackets.add(packet);
+            } else {
+                log.warn("Discovery packet was rejected for {} id:{} task:{}", endpoint, packetNo, taskId);
```

Comment:
add more detail message.

Reply:
More detailed message is logged by floodlight.